shared_infra/dynamo_write_heartbeat/iam_role_policy.tf

```diff
@@ -0,0 +1,11 @@
+data "aws_iam_policy_document" "dynamodb_delete_item_policy" {
+  statement {
+    actions = [
+      "dynamodb:DeleteItem",
+    ]
+
+    resources = [
+      "arn:aws:dynamodb:::",
```

Comment:
I think we should scope this policy more tightly – ideally to the individual tables, or even the specific item this Lambda tries to delete:

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/specifying-conditions.html